Gemfile

```diff
@@ -77,8 +77,8 @@ end
 
 group :development, :test do
   gem "awesome_print",            "~> 1.8", require: "ap"
-  gem "bullet",                   "~> 5.6", ">= 5.6.1"
-  gem "dotenv-rails",             "~> 2.2", ">= 2.2.1"
+  gem "bullet",                   "~> 5.7"
```

Comment:
Yeah, I'm not too familiar with this code, but it seems like bullet is saying we should remove eager loading, which I'm not sure is actually a good idea if the eager loading was added specifically to avoid N+1 queries.

I think we might be seeing this bug: https://github.com/flyerhzm/bullet/issues/147
What do you think?

Reply:
I agree looks like the bug you mentioned. Let's wait and see if it gets fixed.

Reply:
@rmacklin Yeah, looking at the code it seems that we do want the eager loading, so unsure why bullet is causing errors there unless it's a bug.

Reply:
Looks like `Bullet.unused_eager_loading_enable = false` could fix the problem

Reply:
Yeah, I was going to ask if you think we should add that for the time being to get the tests passing

Reply:
Gave this fix a try and unfortunately it didn't fix the build failures. 😕

Reply:
@BenEmdon I tried it as well and it seems to have fixed the failures

Reply:
@johndbritton Does this satisfy your request for changes? If so, can you update your review status?